docs/standards.md

```diff
@@ -4,7 +4,29 @@ Since SG2 is a living design library that is home to reusable patterns, it's imp
 
 Any new work contributed into SG2 must meet the following standards:
 
-### PatternLab Organization
+### Workflow standards for contributing to SG2/D8
+
+#### Keep work discrete
+- Encapsulate your work in the smallest discrete chunks possible.
+- Avoid using `rebase` 
+- Avoid branching off of unmerged branches. Doing so can make your code difficult for others to understand and review, which can impede workflow and make working as a team harder; instead, branch off of `develop` whenever possible.
+- Only work on what's in your assigned ticket. If you find that your ticket requires functionality/styling/etc. that is defined in a different ticket, either coordinate with the developer to whom that ticket's assigned, or switch to working on the ticket if unassigned. - only work on the stuff in your assigned JIRA ticket. If the ticket is not assigned to you but you feel you may need to work on it (or need code from it or whatever) reach out to the developer 
+- What to do if you're blocked? Add the `blocked` label and comments in th PR and in JIRA. Blocked tickets should be linked accordingly in JIRA (if they are not, add the link).
```

Comment:
Also please bring up during scrum. Also if you find a bug or are blocked then notify the PMs in slack via the #dtt-web-dev channel.

MUST be in JIRA ticket
Nice to have is also notify in a PM-friendly slack channel.